common/country_types/hfe_country_types_pirate.txt

```diff
@@ -149,7 +157,7 @@ hfe_pirate = {
 		hostile_when_attacked = yes
 		hostile = yes
 		needs_border_access = no
-		# pirate = yes # wonder what it do
+    	pirate = yes # wonder what it do
```

Comment:
I have no idea what is option do, better to turn off.

Reply:
It's specifically made for pirates. If it interferes or breaks, we can change it later.